### PR TITLE
Add MBOM to part excel import

### PR DESCRIPTION
**Pull request by ConnorWasHere**
* Use queue structure to calculate mbom tree from depth column

### CODE REVIEW

File: importers/inventory/excel/import.py [hunk 1]
```diff
@@ -97,6 +97,34 @@ def _bulk_create_part_inventories(api: Api, df: pd.DataFrame, parts: dict,
     return True
 
 
+def _create_mbom(api: Api, mbom_struct: dict, parts: dict) -> bool:
+    """
+    Batch create MBOM items for every part in the import excel.
+
+    Args:
+        api (Api): API instance to send authenticated requests
+        mbom_struct (dict): Mapping of part number to parent part and quantity
```
Comment (ktal90) [inline importers/inventory/excel/import.py:106]: Is this the best name for this? The "struct" name is a data structure in other languages. It kind of confuses on the real type (dict). Maybe `mbom_map` or `mbom_qty_map`? What do you think?

File: importers/inventory/excel/import.py [hunk 2]
```diff
@@ -97,6 +97,34 @@ def _bulk_create_part_inventories(api: Api, df: pd.DataFrame, parts: dict,
...
+    mboms = api.send_api_request(create_mutations)
+    for idx, mbom_item in enumerate(mboms):
+        if 'errors' in mbom_item and len(mbom_item['errors']) > 0:
+            logging.warning(mbom_item['errors'][0]['message'])
+    return True
+
+
 def _bulk_create_parts(api: Api, df: pd.DataFrame, parts: dict) -> bool:
     """
     Batch create parts for every row in the import excel.
```
Comment (ktal90) [inline importers/inventory/excel/import.py:130]: Could you extend docstrings here to further explain what this function now takes into consideration with depth and quantities? Also, maybe an example?

**Review by ktal90 — CHANGES_REQUESTED** [review]

File: importers/inventory/excel/import.py (hunk 1 above)
Comment (ConnorWasHere) [reply importers/inventory/excel/import.py:106]: good call! Done

File: importers/inventory/excel/import.py (hunk 2 above)
Comment (ConnorWasHere) [reply importers/inventory/excel/import.py:130]: Done!

**Review by ktal90 — APPROVED** [review]

**Pull request merged**